packages/contracts-rfq/contracts/FastBridgeV2.sol

```diff
@@ -131,28 +159,33 @@ contract FastBridgeV2 is Admin, IFastBridgeV2, IFastBridgeV2Errors {
     }
 
     /// @inheritdoc IFastBridgeV2
+    // TODO: reduce cyclomatic complexity alongside arbitrary call
+    // solhint-disable-next-line code-complexity
     function relay(bytes memory request, address relayer) public payable {
         if (relayer == address(0)) revert ZeroAddress();
```

Comment:
That's a very good point, using a V1 bytes request must end up with a revert here (right now I believe it would not).

Reply:
@parodime actually as seen in 67f2b3083d511ae4cd0e48e61ea59f60c8ca2ea9, this ends up with a revert, although without a specific revert reason (as abi decoding fails). Commented in SLT-273 so that we remember to introduce the explicit revert reason for that once the `BridgeRequestV2` structure is finalised.